### ISSUE TITLE
Make sure the grade is actually a grade.

### ISSUE THREAD

**Issue by Sandeeprds95** [varundey/srmgpacalculator#7]
**1.** Under the grade column, even an integer or other than the actual grade is being considered.
Make sure your 'w' variable is set to 0 if not the above mentioned grades.
Also, if the credits is mentioned and no grade is mentioned, (or) vice-versa, the calculator is still functional, adding some validation will help.

**2.** Tab key on the keyboard doesn't seem to update the 'total' value, only a mouse click does.

**3.** don't think the subject code (or) subject name is as such necessary here. Even though the GPA calculator is operational without the input, I think it's unnecessary.

**All in all, good job!**

**Comment by varundey** [comment]
Thanks for raising an issue.
1. Great input. I am writing a regex function and a validation function :+1: 
2.  That was a minor code glitch due to merging of code with our other contributor. It is rectified now :smiley: 
3. After developing when i began testing it, I found it hard to remember if I already put the marks of a subject ot not. The subject column is just there as a reminder that a particular subject is already entered and is therefore not a mandatory column.

Thanks a lot. I started it off as an idea after watching [The Big Short](http://www.imdb.com/title/tt1596363/) at 4 A.M. :smile:
